CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to Vets.gov
+
+## How to Contribute
+
+There are many ways to contribute to this project:
+
+**Bugs**
+
+If you spot a bug in the source code or a mistake in the documentation, let us know! File a GitHub Issue for this project. When filing an issue add the following:
+
+- Title: Sentence that summarizes the bug concisely
+- Comment:
+    - The environment you experienced the bug (browser, browser version, kind of account any extensions enabled)
+    - The exact steps you took that triggered the bug. Steps 1, 2, 3, etc.
+    - The expected outcome
+    - The actual outcome, including screen shot
+    - (Bonus Points:) Animated GIF or video of the bug occurring
+- Label: Apply the label `bug`
+
+**Code Submissions**
+
+Due to some VA-network constraints and limited team bandwidth, we are not able to review community pull requests in a timely fashion so we ask that you file an issue so that we can work the fix into the work that our team is already doing. 
```

Comment:
Here's my suggestion for wording:

> Due to limited team bandwidth and constraints on the VA, we cannot commit to reviewing community pull requests in a timely fashion. We ask that you file an issue before beginning work on any non-trivial PR, so that we can incorporate the proposed change into the work that our team is already doing.

Is that better?